platforms/ios/lib/WysiwygComposer/Tests/WysiwygComposerTests/Extensions/String+LatinLangugesTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2024 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@testable import WysiwygComposer
+import XCTest
+
+final class StringLatinLangugesTests: XCTestCase {
+    func testLatinLangugeCharacters() throws {
+        XCTAssertTrue("hello".containsLatinAndCommonCharactersOnly)
+        XCTAssertTrue("helló".containsLatinAndCommonCharactersOnly)
+        XCTAssertTrue("helló, ".containsLatinAndCommonCharactersOnly)
+        XCTAssertTrue("helló, ".containsLatinAndCommonCharactersOnly)
+        XCTAssertTrue("😄🛴🤯❤️".containsLatinAndCommonCharactersOnly)
+        XCTAssertTrue(String.object.containsLatinAndCommonCharactersOnly)
```

Comment:
what's this test case about?

Reply:
This is the obeject replacement unicode character as defined [here](https://github.com/matrix-org/matrix-rich-text-editor/blob/main/platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/Extensions/String%2BCharacter.swift#L31). It get's inserted during some of the`shouldChangeTextIn` updates, I think I noticed it during voice dictation so is one we should accept.